src/components/common/Filter.js

```diff
@@ -235,6 +243,7 @@ function Filter(props) {
               </Grid>
 
               <Grid item xs={6}>
+                {/* I think you should use datepickers here :) */}
```

Comment:
Switching the input to a [date-picker](https://mui.com/components/date-picker/)  could be posted as a separate issue